packages/rx_bloc_list/example/pubspec.yaml

```diff
@@ -5,22 +5,27 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.13.0 <3.0.0"
```

Comment:
@DDavidPrime, do we need to constraint the devs to use 2.13.0, or rx_bloc will be running fine with 2.12 as well?